ReactiveCocoa/Swift/Action.swift

```diff
@@ -134,7 +134,7 @@ public final class Action<Input, Output, Error: ErrorType> {
 public final class CocoaAction: NSObject {
 	/// The selector that a caller should invoke upon a CocoaAction in order to
 	/// execute it.
-	public static let selector: Selector = "execute:"
+	public static let selector: Selector = #selector(CocoaAction.execute(_:))
```

Comment:
> Are the changes like these required for Swift 2.2? (Are they errors or just warnings?) 
Most of them are not, they're just warnings.

> They're breaking changes with Swift 2.1 / Xcode 7.2.1.
Oh right, good point... we need to make these changes at some point anyway, so I'd vote for not staying behind. There's already a major version available for `Swift 2.1`.

> so we'll need to release these as RAC 5.0 to follow semver.
That's absolutely true, but I don't think it'll be productive to release a new major version of RAC for every Swift version, which given that it's not ABI stable, is by definition always a breaking change. I'd vote for following the same semantic versioning as `Swift`: since this is supposedly just a minor change, I'd say we release this as `ReactiveCocoa 4.1`, and reserve `ReactiveCocoa 5.0` for `Swift 3.0`. Not that there's anything wrong with upgrading to a new major version, but I think it's really confusing. When it comes to keeping backwards compatibility I just think that we don't have enough time to deal with all the pains involved in that, so I'd strongly suggest looking forward :)